src/Database/SchemaCache.php

```diff
@@ -104,6 +105,7 @@ public function getSchema(Connection $connection)
 
         $config = $connection->config();
         if (empty($config['cacheMetadata'])) {
+            Log::info("Metadata cache was disable in config for `{$connection->configName()}`. Enabling to clear cache.");
```

Comment:
```suggestion
            Log::info("Metadata cache was disabled in config for `{$connection->configName()}`. Enabling to clear cache.");
```

Nitpick: I personally prefer using `sprintf()` instead of doing method calls inside strings.

Reply:
done.